### DISCUSSION TITLE
About unordered list presentation form

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1196] [Ideas]
The current unordered list within the document after rspress build is presented in the form shown below, with no hierarchical differences shown:
<img width="701" alt="now" src="https://github.com/web-infra-dev/rspress/assets/171249651/f3846f18-e104-4039-9273-c9de13009896">
Would you consider referring to the current unordered list presentation form in the feishu documentation?
<img width="203" alt="feishu" src="https://github.com/web-infra-dev/rspress/assets/171249651/4bac4bdb-1f8c-49ed-92b4-e7e684b4845b">

**Comment by Timeless0911** [comment]
The implementation of the two are completely different. Feishu document is an actual DOM structure, while the static site generator uses the `:::marker` pseudo-element. So you can control it by customizing the style in Rspress.

**Reply by momo91377** [reply]
Is it possible to do something in version 2.0 to optimize it, currently we see not only the unordered list, ordered list is also the same display problem. When used nested, it seems to affect the usability.

**Reply by Timeless0911** [reply]
Yes, are you interested in it? You can open a PR as you expected. 

FYI

The list component style is controled by tailwindcss now
https://github.com/web-infra-dev/rspress/blob/040dcb316ca9cc0a5bf282e5cd5d8c788e4de1b7/packages/theme-default/src/layout/DocLayout/docComponents/list.tsx#L3-L9
You can set the `list-style-type` attributes like
```css
ul ul {
    list-style-type: circle;
}
```

**Reply by momo91377** [reply]
Thanks a lot. I'll make an attempt.

**Reply by momo91377** [reply]
Is it only available as a customized component, but not through a custom css file, i try to add it to the css file ,but useless.

**Reply by Timeless0911** [reply]
This maybe related to the css priority after bundle, inline tailwind classname has high priority.

**Reply by momo91377** [reply]
Is there any way to change it?

**Reply by Timeless0911** [reply]
Remove tailwind classname, use css to set it completely.

**Reply by momo91377** [reply]
Got it, thanks
